homework-submission/todo-app/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import Form from './Components/Form';
+import List from './Components/List';
+import './App.css';
+
+@inject('TasksStore')
+@observer
+
+class App extends Component {
+  componentWillMount() {
```

Comment:
Excellent Use of `componentWillMount`, But this lifeCycle has been deprecated, you can use `componentDidMount` instead or the `constructor` if you wanted it before the mounting Cycle.